mobile/src/bundles/common/components/input/input.tsx

```diff
@@ -12,12 +12,19 @@ import { useFormController } from '~/bundles/common/hooks/hooks';
 
 import { styles } from './styles';
 
+type InputType = 'text' | 'number' | 'email' | 'password';
+
 type Properties<T extends FieldValues> = {
     control: Control<T, null>;
     errors: FieldErrors<T>;
     label: string;
     name: FieldPath<T>;
     placeholder: string;
+    type: InputType;
+    secureTextEntry?: boolean;
+    editable?: boolean;
+    multiline?: boolean;
+    numberOfLines?: number;
```

Comment:
```suggestion
...props
```

Reply:
The variable `props` should be named `properties`. A more descriptive name will do too.eslint[unicorn/prevent-abbreviations](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v48.0.1/docs/rules/prevent-abbreviations.md)

I recieved such suggestion from eslint, so should I use properties as variable or just ignore this?

Reply:
Merge latest development an error will disappear